gax-grpc/src/main/java/com/google/longrunning/OperationsClient.java

```diff
@@ -109,6 +117,80 @@
   private final OperationsSettings settings;
   private final OperationsStub stub;
 
+  public static class ListOperationsPagedResponse
```

Comment:
I'd prefer these inner classes to go at the end of the file

Reply:
Done.